contracts/governance/FeeSharingProxy/FeeSharingLogic.sol

```diff
@@ -345,11 +412,10 @@ contract FeeSharingLogic is SafeMath96, IFeeSharingProxy, Ownable, FeeSharingPro
 
         /// @dev Additional bool param can't be used because of stack too deep error.
         if (_maxCheckpoints > 0) {
+            if (start >= numTokenCheckpoints[_loanPoolToken]) {
```

Comment:
pls consider renaming `start` to `processedCheckpoints` which it essentially is  
else it `start` should be `processedCheckpoints[_user][_loanPoolToken] + 1`

Reply:
updated in this [commit](https://github.com/DistributedCollective/Sovryn-smart-contracts/pull/458/commits/97072b5fa5c3ba131d4e82392853d22ebbd4b808)